runtime/vm/createramclass.cpp

```diff
@@ -530,8 +540,9 @@ addInterfaceMethods(J9VMThread *vmStruct, J9ClassLoader *classLoader, J9Class *i
 							existing_entry = previous_entry;
 							J9EquivalentEntry * new_entry = (J9EquivalentEntry*) pool_newElement(equivalentSets);
 							if (NULL == new_entry) {
+								/* OOM will be thrown */
 								setNativeOutOfMemoryError(vmStruct, 0, 0);
```

Comment:
`setNativeOutOfMemoryError` should this have been removed like the call above?

Reply:
Arg, yep.

Reply:
Fixed.